packages/node_modules/@node-red/editor-client/src/js/validators.js

```diff
@@ -13,23 +13,53 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
+
 RED.validators = {
-    number: function(blankAllowed,mopt){
+    number: function(options,mopt) {
+        const DEFAULTS = {
+            blankAllowed: false,
+        }
+        // Also supports:
+        // - '>' : greater than
+        // - '>=' : greater than or equal to
+        // - '<' : less than
+        // - '<=' : less than or equal to
+        if (typeof options === 'boolean') {
+            options = {
+                blankAllowed: options
+            }
+        }
+        options = Object.assign(DEFAULTS, options)
+
         return function(v, opt) {
```

Comment:
```
        /**
         * Validator function for number values.
         * @param {string} v - The value to validate.
         * @param {Object} [opt] - Optional object
         * @param {string} [opt.label] - Label for the value, used in error messages. (optional)
         * @return {boolean|string} Returns true if the value is valid, or an error message if invalid.
         */
         return function(v, opt) {
```

Please consider adding type hint

Sorry, GH wouldnt let me make this untouched line a suggestion